src/pages/AddJob.jsx

```diff
@@ -42,6 +43,73 @@ export default function AddJob() {
     }
   );
 
+  useEffect(() => {
+    let hasdraft = window.localStorage.getItem("draft");
+    if (hasdraft !== null && toastid.length === 0) {
+      toastid.push(1);
+      toast(
+        (t) => (
+          <div className="flex flex-row justify-between items-center w-[350px] p-4 rounded-lg shadow-md">
+            <div className="w-1/2">
+              <p>Load draft?</p>
+            </div>
+            <div className="flex flex-row w-1/2 justify-between h-full">
+              <div>
+                <button
+                  onClick={() => {
+                    loadDraft();
+                    setHasLoad(true);
```

Comment:
you are not using the value `hasLoad`.  no need to set the value. remove all things for this state

Reply:
I'm using hasLoad in submit onclick. The meaning is to check if the employer has used the draft to submit posting.
If the employer has saved a draft and then create a new job instead of using the draft, the draft should not be deleted.

Reply:
it does not matter if you have the draft or not, just clear the certain item when you submit